### ISSUE TITLE
Failing installation of SixLabors.ImageSharp in NuGet VS 15.4.2

### ISSUE THREAD

**Issue by krystyna93** [aspnet/Home#2267]
I have been having issues with the installation of the SixLabors.ImageSharp package.
I have tried to repair VS 15.4.0 but the installation still failed.
I have updated to Visual Studio 15.4.2, but I am still getting the same error
I have also uninstalled the package, rebooted VS and reinstalled the package again numerous time, but to no avail.
I have also uninstalled and reinstalled the Microsoft.NetCoreApp.All package, but that did not work either.

**ERROR LIST window :**

Error Package restore failed. Rolling back package changes for 'myprojectname'.

**I am also receiving the following error in my OUTPUT window :**

    Restoring packages for C:\Users\Me\Source\Repos\project\src\project\project.csproj...
    Package SixLabors.ImageSharp 1.0.0-beta0001 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0). Package SixLabors.ImageSharp 1.0.0-beta0001 supports:

    netstandard1.1 (.NETStandard,Version=v1.1)
    netstandard1.3 (.NETStandard,Version=v1.3)
    Package restore failed. Rolling back package changes for 'project'.
    Time Elapsed: 00:00:01.1947210
    ========== Finished ==========

I dont know what could be the issue, am I missing some library for it?
Can someone please enlighten me, I cant seem to find a solution to this issue, which has been bugging me all day

**Comment by lillo42** [comment]
Your problem seems to be SixLabors.ImageSharp, This lib is not compatible with .NET Core 2.0, try to update this package or change to another lib.

**Comment by krystyna93** [comment]
Hi Joeaudette, thanks for your reply. I did not think it was an error in relation to the netcoreapp2.0 net standard class either.

I do have a small breakthrough though: I went to the tools tab>options>nuget package manager, and selected the 'clear nuget cache/s', which cleared the cache. Then I rebooted VS up, and tried installing the sixlabors.imagesharp package for this test solution, and it surprisingly WORKED!

However, when I tried to do the same in another project solution of mine, it said that access was denied for another package of mine 'build bundler minifier', when I tried to clear the nuget cache here! 
I am not sure if I need to remove the 'build bundler minifier' package first so that I can reinstall the imagesharp package first, then reinstall the build bundler minifier.

Any thoughts?

**Comment by joeaudette** [comment]
@krystyna93 every now and then I have seen stubborn nuget cache cause problems, you can clear it manually by deleting files under c:\Users\yourusername\.nuget\packages then try from the command line in the solution folder dotnet restore --no-cache

**Comment by krystyna93** [comment]
Ah, I will definitely try that, @joeaudette .
Thanks a lot for that. I will test this out later today.
Cheers! 👍

**Comment by krystyna93** [comment]
@joeaudette 
Well I cleared the NuGet cache, and had to manually remove the BuildBundlerMinifier first, because it was persistently stubborn and would not delete. I reinstalled the SixLabors.ImageSharp package, and then reinstalled the BuildBundlerMinifier, and now it works again!

I didn't have to use the command line at all to clear the cache.

I think it should be recorded for others, that any issues regarding NuGet package installation errors/issues could indeed be related to the cache folder/s.

Cheers!
